2.7 Docs Review

jamesnocentini opened the pull request:
[Mercury Documentation Plan](https://hub.internal.couchbase.com/confluence/display/techpubs/Mercury+Documentation+Plan)

jamesnocentini committed:
bump version

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

rename java.adoc to java-android.adoc

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add DB001 content

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

rename android project to java-android

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add java-platform example project

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add CBL dependency

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

clean up asciidoc attributes

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add Examples file

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add first java snippet

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

update asciidoc attributes

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

copy ascii content from java android

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

add image and re-work section

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

- add page aliase
- rename java.adoc to java-platform.adoc

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

- remove callout

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

- rename snippet-java attribute to snippet-java-platform

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

- remove Logging [DEPRECATED] section

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5710

- update initializer snippet

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5909

jamesnocentini committed:
https://issues.couchbase.com/browse/DOC-5852

ibsoln commented on modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Examples.java:
```diff
@@ -0,0 +1,92 @@
+        System.out.println("Learning " + document.getString("language"));
+
+        // Create a query to fetch documents of type SDK.
+        Query query = QueryBuilder.select(SelectResult.all())
```
```suggestion :: 

The following query will list the existing documents; enabling easy review of server changes being replicated to client

`        Query query2 = QueryBuilder.select(SelectResult.expression(Meta.id),
                        SelectResult.property("language"),
                        SelectResult.property("version"))
                .from(DataSource.database(database))
                .where(Expression.property("type").equalTo(Expression.string("SDK")))
                .orderBy(Ordering.expression(Meta.id));
        try {
            ResultSet rs2 = query2.execute();
            for (Result result : rs2) {
                System.out.println("Student ID :: " + result.getString("id"));
                System.out.println("Learning :: " + result.getString("language") + " version :: "+ result.getFloat("version"));
            }
        } catch (CouchbaseLiteException e) {
            System.out.println(e.getLocalizedMessage());
        }
`

 ```

ibsoln reviewed (commented):
test

ibsoln commented on modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Main.java:
```diff
@@ -0,0 +1,8 @@
+    public static void main(String[] args) {
+	// write your code here
+    }
+}
```
@ibsoln TODO?

ibsoln committed:
DOC-5709-RelNotes-CBL (#146)

https://issues.couchbase.com/browse/DOC-5709

Add CBL release notes.

Also added in some standard attributes

ibsoln committed:
Doc 5710 rename java platform java (#149)

* DOC-5710-Rename-JavaPlatform-Java

Trial commit to rename Java Platform as Java and Java (Android)

* DOC-5710-Rename-JavaPlatform-Java

Revert 
modules/ROOT/examples/java-android/app/gradle/wrapper/gradle-wrapper.properties 
changes to.4.4-all.zip

* DOC-5710-Rename-JavaPlatform-Java

ibsoln reviewed (changes requested):
I don't understand this PR#133. It seems to be pushing 2.7 changes back into Release 2.6 and I don't believe that is right? 

Can you look at this for me please? @RichardSmedley

RichardSmedley commented:
Looks like James made this against 2.6 in error - easily done in the GH ui.
I'm closing it.

ibsoln commented:
Thanks Richard


From: Richard Smedley <notifications@github.com>
Reply-To: couchbase/docs-couchbase-lite <reply@reply.github.com>
Date: Thursday, 19 December 2019 at 11:35
To: couchbase/docs-couchbase-lite <docs-couchbase-lite@noreply.github.com>
Cc: "Ian Bridge [C]‌" <ian.bridge@couchbase.com>, Mention <mention@noreply.github.com>
Subject: Re: [couchbase/docs-couchbase-lite] 2.7 Docs Review (#133)


Looks like James made this against 2.6 in error - easily done in the GH ui.
I'm closing it.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/couchbase/docs-couchbase-lite/pull/133?email_source=notifications&email_token=AMSVPQXJWJK6IEPW5YE6GD3QZNL7XA5CNFSM4JB4RQD2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHJKQ2I#issuecomment-567453801>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMSVPQXOKBHMEJ22FORWOVTQZNL7XANCNFSM4JB4RQDQ>.
